packages/flutter/test/material/time_picker_test.dart

```diff
@@ -222,6 +222,64 @@ void main() {
     expect(selectedLabels.map<bool>((dynamic tp) => tp.inner as bool), inner0To23);
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/164860
+  testWidgets('Material3 - formats 24-hour numbers correctly in Farsi', (
+    WidgetTester tester,
+  ) async {
+    await mediaQueryBoilerplate(
+      tester,
+      locale: const Locale('fa', 'IR'),
+      materialType: MaterialType.material3,
+    );
+
+    final List<String> labels00To23 = <String>[
```

Comment:
Does it mean all numbers are displayed in Arabic numerals? Although this is consistent, I thought https://github.com/flutter/flutter/issues/164860 wanted all numbers to display in Farsi. (I'm not really sure which is the correct)

Reply:
I confirm that numbers should be in Farsi, according to Android Google Calendar app.

![image](https://github.com/user-attachments/assets/de9e91cc-dbb9-47d6-acff-c418c6297ff6)

Reply:
Oh, sorry. My bad, even at top I could see its farsi. Let me do the change and push.